CONTRIBUTING.md

```diff
@@ -7,30 +7,46 @@ If you just want to contribute a client library in a new language and not make c
 ## Requirements
 
 - MacOS or Linux (Windows may work too, but we haven't tested it)
-- NodeJS 12.x or 14.x
-- Yarn
-- Python 3.6+ (for the stats engine)
-  - scipy
-  - numpy
-  - pandas
-- Docker (for running MongoDB locally)
+- [NodeJS](https://nodejs.org/en/download/package-manager/) 12.x or 14.x
+  - Check version by running `node -v` on terminal
+- [Yarn](https://classic.yarnpkg.com/en/docs/install#windows-stable) 
```

Comment:
This is the windows-specific installation page.  If you get rid of `#windows-stable` at the end, it will auto-detect the operating system and redirect to the correct URL.

Reply:
Updated and changed on this branch